src/content_cao.cpp

```diff
@@ -308,8 +308,13 @@ class ItemCAO : public ClientActiveObject
 
 	void initialize(const std::string &data);
 
-	aabb3f *getSelectionBox()
-		{return &m_selection_box;}
+
+	virtual bool getSelectionBox(aabb3f *toset){
+		(*toset)=m_selection_box;
```

Comment:
Ditto, also spaces around `=`.
See [the code style guidelines](http://dev.minetest.net/Code_style_guidelines) for more.